<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>814</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>868</width>
    <height>814</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>868</width>
    <height>814</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Source Software Developed by Amr Khalil 2021</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dbTableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>641</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScrollMargin">
     <number>13</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>15</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>147</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>27</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Zeit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Datum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Maske</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Anwesenheitstabelle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tabelle aktualisieren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>750</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesichtserkennung</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="userPhoto1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>549</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userPhoto2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>660</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userName2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>750</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="userName1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>181</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="detectButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gesicht erkennen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adduserButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+ / - Gesicht</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nameButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="maskButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mundschutz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="genderButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Geschlecht</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ageButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Alter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emotionButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Emotion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="faceLandmarksButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gesichtsmerkmalen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="blurButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Verschwimmen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>38</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="startButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Webcam starten</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Kamera">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kamera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="openVideoButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Video / Bild  öffnen</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="saveimageButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bild speichern</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bekannte Gesichter</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>750</y>
      <width>171</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="count">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Anzahl:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
